test/test_torch.py

```diff
@@ -2386,39 +2386,259 @@ def _test_rpow(self, cast):
     def test_rpow(self):
         self._test_rpow(self, lambda x: x)
 
+    @staticmethod
+    def _check_against_np(self, tensor, exp):
+        tensor_np = tensor.cpu().numpy()
+        exp_np = exp if isinstance(exp, int) else exp.cpu().numpy()
+        expected = torch.tensor(tensor_np ** exp_np, dtype=tensor.dtype)
+        self.assertEqual(torch.pow(tensor, exp), expected)
```

Comment:
Will fail cuda test as comparison between cuda and cpu tensors is not allowed

Reply:
That's not true, [assertEqual moves their arguments to the same device](https://github.com/pytorch/pytorch/blob/master/test/common_utils.py#L626)

Reply:
assertEqual does some funny things... it should really check the device and dtype (it doesn't) but that is unrelated to this PR